### PR TITLE
Build with docker

### LINKED ISSUE #27: add docker files for different components of the application
- Application
- PostgreSQL
- Redis

### PR DESCRIPTION
**Pull request by deepak11627**
Closes #27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ![ddd layers diagram image](./ddd-arch-diag.jpeg)
 
+## Building application 
+Use the `build.sh` executable to bring docker instances for the application and its dependencies with docker-compose.
+```
+./build.sh
```
[the remark this answers is not in the document]
Comment (deepak11627) [reply README.md:8]: not meant to be used for initialising the db. its for the pipeline.

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+GO111MODULE=on CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -a -installsuffix cgo -o app .
+
+docker-compose up -d
+
```
[the remark this answers is not in the document]
Comment (deepak11627) [reply build.sh:6]: not meant to be used for initialising the db. its for the pipeline.

File: docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    image: "redis:alpine" # Use a public Redis image to build the redis service
+    environment:
+      REDIS_PASSWORD: ''
+      REDIS_USER: ''
+    ports:
+      - "6379:6379"
+    networks:
+      - backend
+
+  # PostgreSQL Service
```
[the remark this answers is not in the document]
Comment (deepak11627) [reply docker-compose.yaml:42]: ok

[the remark this answers is not in the document]
Comment (deepak11627) [reply docker-compose.yaml:42]: which db file to be used for postgresql ? I can see quite a few sql files.

[the remark this answers is not in the document]
Comment (danglingmind) [reply docker-compose.yaml:42]: postgres.sql

**Comment by danglingmind** [comment]
already done

**Pull request closed without merging**